pages/web-literacy/activitydata.js

```diff
@@ -0,0 +1,189 @@
+var search = "Search";
```

Comment:
it feels like these are constants, not variables; can we make them all caps to make their role clear?